sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -1066,3 +1066,59 @@ test_suite(
     ],
     tests = ["asm_interrupt_handler_fpga_cw310_test_otp_" + otp_name for otp_name in OTP_CFGS_EXEC_DISABLED],
 )
+
+[opentitan_multislot_flash_binary(
+    name = "boot_policy_valid_img_a_{}_b_{}".format(
+        "empty" if offset == "0x0" else "bad",
+        "empty" if offset != "0x0" else "bad",
+        ),
+    srcs = {
+        ":empty_test_slot_{}_corrupted".format(slot):{
+        "key": "test_key_0",
+        "offset": offset,
+        },
+    },
+    devices = ["fpga_cw310"],
+)for slot, offset in SLOTS.items()]
+
+SIGVERIFY_BAD_CASES = [
+    {
+        "desc": "bad",
+        "suffix": "_corrupted",
+    },
+    {
+        "desc": "empty",
+        "suffix": "_empty",
+    },
+]
+[
+    opentitan_functest(
+        name = "sigverify_always_a_{}_b_{}".format(
+            a["desc"],
+            b["desc"],
+        ),
+        cw310 = cw310_params(
+            exit_failure = MSG_PASS,
+            exit_success = DEFAULT_TEST_FAILURE_MSG,
+        ),
+        key = "multislot",
+        ot_flash_binary = ":boot_policy_valid_img_a_{}_b_{}".format(
+            a["desc"],
+            b["desc"] if a != b and b != "empty" else "bad",
```

Comment:
I don't like it but I could not figure out a better way without code duplication (not using the for loop).

Reply:
no worries, we can clean this up in a second pass.